fastlane/lib/fastlane/actions/hockey.rb

```diff
@@ -33,6 +33,22 @@ def self.upload(api_token, ipa, options)
         end
       end
 
+      def self.upload_dsym(api_token, app_id, bundle_version, options)
```

Comment:
Here the var is called `bundle_version`, but when called it gets `app_version_id` as param. 
Are these two the same thing? Why the different naming?

Reply:
Yes, they are the same. I used the same old name for `PUT` method. I will change that, also I will change `self.upload_dsym` to `self.upload_ipa_dsym` as it's possible to upload IPA and/or dSYM with the same method.